polars/polars-time/src/windows/test.rs

```diff
@@ -690,15 +690,15 @@ fn test_rolling_lookback() {
     )
     .unwrap();
     assert_eq!(dates.len(), groups.len());
-    assert_eq!(groups[0], [0, 5]);
-    assert_eq!(groups[1], [1, 5]);
-    assert_eq!(groups[2], [2, 5]);
-    assert_eq!(groups[3], [3, 5]);
-    assert_eq!(groups[4], [4, 5]);
-    assert_eq!(groups[5], [5, 4]);
-    assert_eq!(groups[6], [6, 3]);
-    assert_eq!(groups[7], [7, 2]);
-    assert_eq!(groups[8], [8, 0]);
+    assert_eq!(groups[0], [1, 4]); // (00:00, 02:00]
```

Comment:
not sure this test was right to begin with

if the dates are:
[00:00, 00:30, ..., 3:30, 4:00]
and the period is `2h` and the offset is `0h`, then the first window is `(00:00, 02:00]`. So that includes points: `[00:30, 01:00, 01:30, 02:00]`, i.e. the starting offset is `1`, and the length is `4`